xml/System.Xml.Schema/XmlSchema.xml

```diff
@@ -68,8 +68,8 @@
  ]]></format>
     </remarks>
     <related type="Article" href="https://msdn.microsoft.com/library/bbbcc70c-bf9a-4f6a-af72-1bab5384a187">Working with XML Schemas</related>
```

Comment:
if you want to fix this one: <~/docs/standard/data/xml/working-with-xml-schemas.md>